resources/references/storefront-reference/twig-function-reference.md

```diff
@@ -32,4 +32,5 @@ In Shopware, we extend Twig's functionality by custom ones. See our own actions
 | `seoUrl()` | Returns seo URL of given route | --- |
 | `searchMedia()` | Resolves media ids to media objects | See [Add media](../../../guides/plugins/plugins/storefront/use-media-thumbnails.md) guide for details. |
 | `rawUrl()` | Returns full URL | --- |
+| `entities()` | Returns entity collection includes associations from a given entity name and array of entity id | --- |
```

Comment:
can you please add a more detailed guide on how to use that function, with some examples on how to use it, what are the ecpected parameters etc.